lib/src/components/NxNexusPageHeader/NxNexusPageHeader.scss

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-page-header--nexus {
+  .nx-page-header__inner {
+    padding-left: 0;
+  }
+
+  .nx-product {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 72px 16px 1fr;
+    grid-template-columns: 72px 1fr;
+    -ms-grid-rows: 39px 0 auto;
+    grid-template-rows: 39px auto;
+    gap: 0 16px;
+    grid-template-areas:
+      "logo top-row"
+      "logo bottom-row";
+
+    height: 72px;
+    margin-bottom: 0;
+  }
+
+  .nx-product__top-row {
+    -ms-grid-row-align: start;
+    align-self: start;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 1;
+    -ms-grid-column: 3;
+    grid-area: top-row;
+    padding-top: $nx-spacing-xs;
+  }
+
+  .nx-product__bottom-row {
+    -ms-grid-row: 2;
+    -ms-grid-row-span: 1;
+    -ms-grid-column: 3;
+    -ms-grid-row-align: start;
+    align-self: start;
+    grid-area: bottom-row;
+    padding-bottom: $nx-spacing-xs;
+  }
+
+  .nx-product__logo {
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 2;
+    -ms-grid-column: 1;
+    grid-area: logo;
+  }
+
+  .nx-product__link {
+    display: block;
+    height: 72px;
+  }
+
+  .nx-product__brand {
+    @include bold();
+    color: #2c2c2c;
+    display: inline-block;
+    font-size: 26px;
+    margin: 0;
+  }
+
+  .nx-product__name {
+    color: #2c2c2c;
+    display: inline-block;
+    font-size: 26px;
+    margin: 0 0 0 6px;
+
+    &::before {
+      content: "";
+      margin-right: 0;
+    }
+  }
+
+  .nx-product__byline, .nx-product__version {
+    color: #474747;
+    display: inline-block;
+    font-size: $nx-font-size-xs;
+    margin: 0;
+  }
+
+  .nx-product__version {
+    margin: 0 0 0 6px;
+
+    &::before {
+      content: "|";
+      display: inline-block;
+      font-size: 12px;
+      margin-right: $nx-spacing-xs;
+    }
+  }
+
+  .nx-page-header__link {
+    color: #2c2c2c;
+    padding-bottom: $nx-spacing-xxs;
+
+    &:active, &.nx-page-header__link--current {
+      border-bottom-color: #00a1e2;
+    }
+  }
+
+  .nx-page-header__links {
+    flex-grow: 1;
```

Comment:
Doesn't seem to be necessary

Reply:
> Doesn't seem to be necessary

Once was, now isn't: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/3adc468fc17c02341b1bfa6efdc5abaeea26e420